apps/bare-expo/scripts/start-ios-e2e-test.ts

```diff
@@ -211,3 +214,26 @@ async function ensureDirAsync(dirPath: string) {
     }
   }
 }
+
+async function setTargetDevice() {
```

Comment:
```suggestion
async function queryFirstDeviceAsync() {
```

this method does not set anything